packages/demo/src/components/examples/ToastConnectedExamples.tsx

```diff
@@ -20,7 +20,6 @@ export class ToastConnectedExamples extends React.Component<IToastConnectedExamp
         return (
             <div className="mt2">
                 <div className="form-group">
```

Comment:
Same question: would it be a good think to switch to `<Section>`?

Reply:
to be fair, this is the _**nitpick**_ pr :joy: